tools/amp_stt/tag_words.xml

```diff
@@ -0,0 +1,21 @@
+<tool id="vocabulary_tagging" name="Vocabulary Tagging" version="1.0.0">
```

Comment:
Can we name the python and xml files with vocabulary_tagging.py/xml

Reply:
Updated